### DISCUSSION TITLE
Removing multiple <br> from the HTML source

### DISCUSSION THREAD

**Discussion by anjanesh** [tinymce/tinymce#9022] [General]
I'm using OpenAI API to generate some text.
At the end of the prompt is "Return the response in HTML format without multiple paragraph breaks." but I get it the TinyMCE editor as like this :

![image](https://github.com/tinymce/tinymce/assets/85973/bb83bd52-bb4e-4c44-abb9-7b3e1a5623f3)

What am I doing wrong here ?
```
document.getElementById(section).value = body.answer;
tinymce.get(section).setContent(body.answer);
```

**Comment by lostkeys** [comment]
TinyMCE will just insert what it is given. So likely you need to tweak your prompts or do content filtering. We have a premium ChatGPT plugin and it was not easy to get it to reliably return HTML in a good format.

**Reply by anjanesh** [reply]
My bad - I had a `nl2br()` post ChatGPT response which was causing the multiple breaks.
